JavaScript/openai-assistants-api/README.md

```diff
@@ -0,0 +1,62 @@
+<!-- banner start -->
+<p align="center">
+  <img src="https://app.autoblocks.ai/images/logo.png" width="300px">
+</p>
+
+<p align="center">
+  📚
+  <a href="https://docs.autoblocks.ai/">Documentation</a>
+  &nbsp;
+  •
+  &nbsp;
+  🖥️
+  <a href="https://app.autoblocks.ai/">Application</a>
+  &nbsp;
+  •
+  &nbsp;
+  🏠
+  <a href="https://www.autoblocks.ai/">Home</a>
+</p>
+<!-- banner end -->
+
+<!-- getting started start -->
+
+## Getting started
+
+- Sign up for an Autoblocks account at https://app.autoblocks.ai
+- Grab your Autoblocks ingestion key from https://app.autoblocks.ai/settings/api-keys
+- Grab your OpenAI API key from https://platform.openai.com/account/api-keys
+- Create a file named `.env` in this folder and include the following environment variables:
+
+```
+OPENAI_API_KEY=<your-api-key>
+AUTOBLOCKS_INGESTION_KEY=<your-ingestion-key>
+```
+
+<!-- getting started end -->
+
+## Install Dependencies
+
+```
+npm install
+```
+
+## Run the script
+
+Without Prompt SDK:
+
+```
+npm run start
+```
+
+With Prompt SDK:
+
+```
+npm run start-with-prompt-sdk
```

Comment:
i think you need to add a step about setting the autoblocks api key, otherwise it'll fail when you try to generate the types for the prompts

Reply:
hmm let me update the make readmes script, cuz it's failing here now

Reply:
updated lmk what you think